src/parsers/psmt2_to_alt_ergo.ml

```diff
@@ -401,8 +423,8 @@ module Translate = struct
     | Cmd_SetLogic _ -> not_supported "set-logic"; acc
     | Cmd_SetOption _ -> not_supported "set-option"; acc
     | Cmd_SetInfo _ -> not_supported "set-info"; acc
-    | Cmd_Push _ -> not_supported "push"; assert false
-    | Cmd_Pop _ -> not_supported "pop"; assert false
+    | Cmd_Push n -> (translate_push n (pos command)) @ acc
+    | Cmd_Pop n -> (translate_pop n (pos command)) @ acc
```

Comment:
- you can have a single function `translate_push_pop` that takes a function as a parameter (`mk_push` or `mk_pop`)
- `acc` could be given as an argument to `translate_xxx` to avoid lists concatenation (`@`)

Reply:
translate_push_pop adds only command now, we avoid lists concatenation